models/experimental/mistral/tt/mistral_attention.py

```diff
@@ -219,7 +219,7 @@ def forward(
         if self.args.FALLBACK_SOFTMAX:
             scores = fallback_ops.softmax(scores, dim=-1)
         else:
-            scores = tt_lib.tensor.softmax(scores, output_mem_config=self.args.out_mem_config)
+            scores = ttnn.softmax(scores, output_mem_config=self.args.out_mem_config)
```

Comment:
We comply with `memory_config` instead of `output_mem_config` in ttnn operations. Please update everywhere

Full list of replacements is in the ticket and includes output_dtype -> dtype and more.